src/core/Akka/Actor/ActorCell.FaultHandling.cs

```diff
@@ -215,14 +215,15 @@ private void Terminate()
             // stop all children, which will turn childrenRefs into TerminatingChildrenContainer (if there are children)
             StopChildren();
 
-            //TODO: Implement when we have ActorSystem.Abort
-            //    if (systemImpl.aborting) {
-            //      // separate iteration because this is a very rare case that should not penalize normal operation
-            //      children foreach {
-            //        case ref: ActorRefScope if !ref.isLocal ⇒ self.sendSystemMessage(DeathWatchNotification(ref, true, false))
-            //        case _                                  ⇒
-            //      }
-            //    }
+            if (SystemImpl.Aborting)
```

Comment:
This is "safe" to do because the `RemoteWatcher` will kill the remotely deployed children eventually.